feg/gateway/docker/python/Dockerfile

```diff
@@ -102,17 +113,17 @@ RUN curl -sSL https://get.docker.com/ > /tmp/get_docker.sh && \
     sh /tmp/get_docker.sh && \
     rm /tmp/get_docker.sh
 
-# Install python code.
-COPY orc8r/gateway/python /tmp/orc8r
-RUN python3.8 -m pip install --no-cache-dir /tmp/orc8r
-
-# update aioh2 since there is no pushed package
-RUN pip3 install --force-reinstall git+https://github.com/URenko/aioh2.git
-
 # Copy the build artifacts.
-COPY --from=builder /build/python/gen /usr/local/lib/python3.8/dist-packages/
+COPY --from=builder /magma/bazel-bin/lte/gateway/release/feg_python_executables.tar.gz \
+                    /tmp/feg_python_executables.tar.gz
+RUN tar -xf /tmp/feg_python_executables.tar.gz --directory / && \
+    rm /tmp/feg_python_executables.tar.gz
+
+WORKDIR /usr/local/bin
 
-# Copy the configs.
+# Copy the configs to overwrite the LTE configs.
+# WARNING: With the current setup this needs to happen
+# after the installing of the Bazel build artifacts!
```

Comment:
WARNING: With the current setup this needs to happen after ~the~ installing ~of~ the Bazel build artifacts!